ding2.info

```diff
@@ -1,5 +1,6 @@
 name = Ding 2
 description = Install a Ding based library site.
+version = 7.x-2.5.1
```

Comment:
Please remove this from the pull request.

[We set `version` during the release process](https://github.com/ding2/ding2/blob/master/docs/release_process.md).